src/controllers/itemDetails/index.js

```diff
@@ -1261,11 +1261,11 @@ function renderTags(page, item) {
     }
 
     for (let i = 0, length = tags.length; i < length; i++) {
-        tagElements.push(tags[i]);
+        tagElements.push('<a href="#/search.html?query=' + encodeURIComponent(tags[i]) + '" class="button-link emby-button" is="emby-linkbutton">' + tags[i] + '</a>');
     }
 
     if (tagElements.length) {
-        itemTags.innerText = globalize.translate('TagsValue', tagElements.join(', '));
+        itemTags.innerHTML = globalize.translate('TagsValue', tagElements.join(', '));
```

Comment:
nitpick: This could open the path for some XSS, though I do not believe it is a realistic threat as the malicious payload would need to be in the tag

Reply:
This should be fine since the tag is url encoded.